pkg/portal/graphql/viewer.go

```diff
@@ -38,6 +61,9 @@ var nodeViewer = node(
 			"geoIPCountryCode": &graphql.Field{
 				Type: graphql.String,
 			},
+			"isOnboardingSurveyCompleted": &graphql.Field{
```

Comment:
Add a field specific resolver here instead of modifying the user model:

```
Resolve: func(p graphql.ResolveParams) (interface{}, error) {
	source := p.Source.(*model.User)
	gqlCtx := GQLContext(p.Context)
	// ...
	return isOnboardingSurveyCompleted, nil
},
```